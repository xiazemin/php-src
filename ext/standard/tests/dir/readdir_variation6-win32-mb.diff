006+ string(65) "私はガラスを食べられますreaddir_variation61.tmp"
007+ string(65) "私はガラスを食べられますreaddir_variation62.tmp"
008+ string(65) "私はガラスを食べられますreaddir_variation63.tmp"
006- string(59) "私はガラスを食べられますreaddir_variation61.tmp"
007- string(59) "私はガラスを食べられますreaddir_variation62.tmp"
008- string(59) "私はガラスを食べられますreaddir_variation63.tmp"
013+ string(65) "私はガラスを食べられますreaddir_variation61.tmp"
014+ string(65) "私はガラスを食べられますreaddir_variation62.tmp"
015+ string(65) "私はガラスを食べられますreaddir_variation63.tmp"
013- string(59) "私はガラスを食べられますreaddir_variation61.tmp"
014- string(59) "私はガラスを食べられますreaddir_variation62.tmp"
015- string(59) "私はガラスを食べられますreaddir_variation63.tmp"