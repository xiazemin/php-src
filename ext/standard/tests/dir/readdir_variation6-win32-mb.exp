*** Testing readdir() : usage variations ***

-- Reading Directory Contents with Previous Handle --
string(1) "."
string(2) ".."
string(59) "私はガラスを食べられますreaddir_variation61.tmp"
string(59) "私はガラスを食べられますreaddir_variation62.tmp"
string(59) "私はガラスを食べられますreaddir_variation63.tmp"

-- Reading Directory Contents with Current Handle (no arguments supplied) --
string(1) "."
string(2) ".."
string(59) "私はガラスを食べられますreaddir_variation61.tmp"
string(59) "私はガラスを食べられますreaddir_variation62.tmp"
string(59) "私はガラスを食べられますreaddir_variation63.tmp"
===DONE===