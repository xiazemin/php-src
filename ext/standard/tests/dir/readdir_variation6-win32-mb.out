*** Testing readdir() : usage variations ***

-- Reading Directory Contents with Previous Handle --
string(1) "."
string(2) ".."
string(65) "私はガラスを食べられますreaddir_variation61.tmp"
string(65) "私はガラスを食べられますreaddir_variation62.tmp"
string(65) "私はガラスを食べられますreaddir_variation63.tmp"

-- Reading Directory Contents with Current Handle (no arguments supplied) --
string(1) "."
string(2) ".."
string(65) "私はガラスを食べられますreaddir_variation61.tmp"
string(65) "私はガラスを食べられますreaddir_variation62.tmp"
string(65) "私はガラスを食べられますreaddir_variation63.tmp"
===DONE===